app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsActivity.java

```diff
@@ -1599,15 +1599,16 @@ public void onResponse(UserItemDataDto response) {
     }
 
     protected void play(final BaseItemDto item, final int pos, final boolean shuffle) {
+        boolean shuffleQuery = item.getBaseItemType() == BaseItemType.MusicAlbum || item.getBaseItemType() == BaseItemType.MusicArtist || item.getBaseItemType() == BaseItemType.Audio ? false : shuffle;
```

Comment:
Can this be moved into it's own private method? That way the logic can be broken down and documented without causing `play` to grow

Reply:
I went ahead and removed all the uses of `shuffleQuery`.
Instead I just removed the use of `shuffle` for creating the queries for `MusicAlbum` & `MusicArtist` in `getItemsToPlay`. If shuffling audio queues locally is accepted, leaving `shuffle` in as an option for the queries, and working around it externally, is messy.